src/application.rs

```diff
@@ -48,6 +50,7 @@ impl Application {
         let terminal = TerminalWrapper::open_terminal()?;
         let mut ui = Ui::new(action_tx.clone(), terminal)?;
         let model = Rc::new(RefCell::new(MonitorModel::default()));
```

Comment:
Why Rc<RefCell> can't it be just Rc<MonitorModel> ?

Reply:
@uncleDecart no, we need interior mutability here or we cannot pass model to other functions because 'it is already borrowed as mut'.